app/src/main/java/com/example/tung/moviedb_07/data/model/MovieList.java

```diff
@@ -1,90 +1,49 @@
 package com.example.tung.moviedb_07.data.model;
 
-import android.os.Parcel;
-import android.os.Parcelable;
-import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
+import java.util.ArrayList;
```

Comment:
ERROR: Unused import: java.util.ArrayList. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports